### PR TITLE
python base additions

### PR DESCRIPTION
**Pull request by EckyYakov**
Made a python base ubuntu image in order to include all of the robot-framework components, there is a python 2 and python 3 version of this image. robot-base and napalm-base use this image as they both require python.

### CODE REVIEW

File: robot-base/Dockerfile.base [hunk 1]
```diff
@@ -1,8 +1,4 @@
-ARG PYTHON_VERSION
-FROM python:"$PYTHON_VERSION"-alpine
-
-RUN apk add --update --no-cache bash sudo su-exec && \
-  rm -rf /var/cache/* /tmp/* /var/log/* ~/.cache
+FROM quay.io/charter-ctec/python-ubuntu-base:latest
```
Comment (v1k0d3n) [inline robot-base/Dockerfile.base]: try not to use latest (this is used in a couple of places). pin it to a specific version for best practice.

**Review by v1k0d3n — COMMENTED** [review]
one nit. once that's changed, this lgtm.

File: robot-base/Dockerfile.base (hunk 1 above)
Comment (EckyYakov) [reply robot-base/Dockerfile.base]: 👍 Will do.

**Comment by EckyYakov** [comment]
Okay. Should be good now. Let me know if anything else gets your attentention

**Review by v1k0d3n — APPROVED** [review]
very clean. nice job.

**Pull request merged**